lightbug_http/http.mojo

```diff
@@ -190,8 +196,8 @@ struct HTTPResponse(Response):
     fn __init__(inout self, body_bytes: Bytes):
         self.header = ResponseHeader(
             200,
-            bytes("OK"),
-            bytes("application/octet-stream"),
+            OK_MESSAGE,
+            "application/octet-stream".as_bytes_slice(),
```

Comment:
we can make this an alias as well like with text/plain

Reply:
Ah yeah, I missed this one, I'll add it